ajax/impact.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2019 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+const DELTA_ACTION_ADD    = 1;
+const DELTA_ACTION_UPDATE = 2;
+const DELTA_ACTION_DELETE = 3;
+
+$AJAX_INCLUDE = 1;
+include ('../inc/includes.php');
+
+// Send UTF8 Headers
+header("Content-Type: application/json; charset=UTF-8");
+Html::header_nocache();
+
+Session::checkLoginUser();
+
+switch ($_SERVER['REQUEST_METHOD']) {
+   // GET request: build the impact graph for a given asset
+   case 'GET':
+      $itemtype =  $_GET["itemtype"]  ?? "";
+      $items_id   =  $_GET["items_id"]    ?? "";
+
+      // Check required params
+      if (empty($itemtype) || empty($items_id)) {
+         http_response_code(400);
+         die;
```

Comment:
do not just die, but die with a return code > 0. Also, you should add a log entry explaining what went wrong

Reply:
Fixed in latest commit.